wcbench.sh

```diff
@@ -159,12 +162,25 @@ install_cbench()
 
     # Install required packages
     echo "Installing CBench dependencies"
-    if "$VERBOSE" = true; then
-        sudo yum install -y net-snmp-devel libpcap-devel autoconf make automake libtool libconfig-devel git
+    
+    if [ "$OS" == 'fedora']; then   #Please check if "echo $OSTYPE" returns "fedora". If not replace "fedora" here with the returned name
+        if "$VERBOSE" = true; then
+            sudo yum install -y net-snmp-devel libpcap-devel autoconf make automake libtool libconfig-devel git
+        else
+            sudo yum install -y net-snmp-devel libpcap-devel autoconf make automake libtool libconfig-devel git &> /dev/null
+        fi
+
+    elif [ "$OS" == 'linux-gnu' ]; then
```

Comment:
This is what Fedora OSs return, need a better check or to manually set the flag.